react/components/OrderContext.tsx

```diff
@@ -0,0 +1,7 @@
+import { createContext, useContext } from 'react'
+
+export const OrderContext = createContext<Order>({} as Order)
```

Comment:
why are you doing `Casting` and then doing `Assertion` on the context object?

Reply:
Because I wrote first the assertion and then the casting 😁 I need to provide a `defaultValue` and ts don't match `{}` as an `Order`.

Reply:
I could remove the casting though